tests/test_run_all_tests.py

```diff
@@ -0,0 +1,300 @@
+    "validmind.data_validation.PiTPDHistogram",
+    # Fix X_train["probability"] to use probability_column
+    "validmind.model_validation.statsmodels.PDRatingClassPlot",
+    # The tests below only work for statsmodels
+    "validmind.model_validation.statsmodels.FeatureImportanceAndSignificance",
+    "validmind.model_validation.statsmodels.RegressionCoeffsPlot",
+    "validmind.model_validation.statsmodels.RegressionFeatureSignificance",
+    "validmind.model_validation.statsmodels.RegressionModelsCoeffs",
+    # Remove in_sample_datasetsm, out_of_sample_datasets inputs
+    "validmind.model_validation.statsmodels.RegressionModelsPerformance",
+    "validmind.model_validation.statsmodels.ScorecardHistogram",
+    # numpy.linalg.LinAlgError: SVD did not converge
+    "validmind.model_validation.statsmodels.DFGLSArch",
+    "validmind.model_validation.statsmodels.ZivotAndrewsArch",
+    # ValueError: This function does not handle missing values
+    "validmind.model_validation.statsmodels.ResidualsVisualInspection",
+    # statsmodels.tools.sm_exceptions.MissingDataError: exog contains inf or nans
+    "validmind.model_validation.statsmodels.ADF",
+    "validmind.model_validation.statsmodels.ADFTest",
+    "validmind.model_validation.statsmodels.PhillipsPerronArch",
+    # ValueError: cannot convert float NaN to integer
+    "validmind.model_validation.statsmodels.KPSS",
+    # Can only compute partial correlations for lags up to 50% of the sample size
+    "validmind.data_validation.ACFandPACFPlot",
+    # ValueError: No datetime columns found in the dataset
+    "validmind.data_validation.TabularDateTimeHistograms",
+]
+SKIPPED_TESTS = []
+SUCCESSFUL_TESTS = []
+
+# Harcode some tests that require specific inputs instead of trying to
+# guess from tags or task_types
+CUSTOM_TEST_INPUT_ASSIGNMENTS = {
+    "validmind.data_validation.DatasetDescription": "classification",
+    "validmind.data_validation.DatasetSplit": "classification",
+    "validmind.model_validation.ModelMetadata": "classification",
+}
+
+# Some tests require specific configurations. This is either expected and required
+# or we need to fix these tests so they can run with sane defaults
+#
+# Here we assign config param keys to each test and then let the test runner know
+# how to load the config for that test
+TEST_TO_PARAMS_CONFIG = {
+    # TODO: features_pairs should default to all input dataset pairs
+    "validmind.data_validation.BivariateFeaturesBarPlots": "features_pairs_raw",
+    "validmind.data_validation.BivariateHistograms": "features_pairs_raw",
+    "validmind.data_validation.BivariateScatterPlots": "features_pairs_raw",
+}
+
+# Global inputs and configurations for the tests
+TEST_CONFIG = {}
+TEST_INPUTS = {}
+
+
+class TestRunTest(unittest.TestCase):
+    pass
+
+
+def create_unit_test_func(vm_test_id, vm_test_class):
+    def unit_test_func(self):
+        self.assertTrue(
+            hasattr(vm_test_class, "required_inputs"),
```

Comment:
The tests that are implemented using decorator functionality won't have `required_inputs` attributes. How do we handle these?

Reply:
The required inputs for function tests should get those set automatically when they are loaded via `load_test`... It basically just checks if the function arguments contains an input name (model, models, dataset, datasets) and sets those as required inputs if so.